anonympy/pdf/utils_pdf.py

```diff
@@ -8,106 +8,108 @@
+) -> None:
     draw = ImageDraw.Draw(image)
     for box in bbox:
         p0, p1, p2, p3 = box
         draw.rectangle([*p0, *p2], fill="black", outline="black")
 
 
-def find_coordinates_pytesseract(matches: List[str],
-                                 data: dict,
-                                 bbox: list) -> None:
+def find_coordinates_pytesseract(matches: List[str], data: dict, bbox: list) -> None:
     for obj in matches:
-        for idx in range(len(data['text'])):
-            i = data['text'][idx]
-            if i.strip() != '':
-                if obj in i:
-                    x, y, w, h = data['left'][idx], data['top'][idx],\
-                                 data['width'][idx], data['height'][idx]
-                    bbox.append((x, y, w, h))
-
-
-def find_coordinates_easyOCR(pii_objects: List[str],
-                             bounds: list,
-                             bbox: list) -> None:
+        for idx in range(len(data["text"])):
+            i = data["text"][idx].strip()
+            if i and obj in i:
+                x, y, w, h = (
+                    data["left"][idx],
+                    data["top"][idx],
+                    data["width"][idx],
+                    data["height"][idx],
+                )
+                bbox.append((x, y, w, h))
+
+
+def find_coordinates_easyOCR(pii_objects: List[str], bounds: list, bbox: list) -> None:
     for obj in pii_objects:
         for i in range(len(bounds)):
-            if (obj.strip() in bounds[i][1].strip()) and \
-               (len(obj.strip()) > 3 and len(bounds[i][1].strip()) > 3):
+            if (obj.strip() in bounds[i][1].strip()) and (
+                len(obj.strip()) > 3 and len(bounds[i][1].strip()) > 3
+            ):
                 bbox.append(bounds[i][0])
 
 
 def find_emails(text: str, matches: list) -> None:
-    match = re.findall(r'[\w.+-]+@[\w-]+\.[\w.-]+', text)
+    match = re.findall(r"[\w.+-]+@[\w-]+\.[\w.-]+", text)
     matches += match
 
 
 def find_numbers(text: str, matches: list) -> None:
-    match = re.findall(r'[0-9]+', text)
+    match = re.findall(r"[0-9]+", text)
     matches += match
 
 
 def find_months(text: str, matches: list) -> None:
-    match = re.findall(r"(?:January|February|March|April|May|June|July|August|\
-        September|October|November|December)", text)
+    match = re.findall(
+        r"(?:January|February|March|April|May|June|July|August|\
+        September|October|November|December)",
+        text,
+    )
     matches += match
 
 
 def alter_metadata(file_name: str, output_name: str):
-    file_in = open(file_name, 'rb')
-
-    pdf_merger = PdfFileMerger()
-    pdf_merger.append(file_in)
-    pdf_merger.addMetadata({
-        '/Author': 'Unknown',
-        '/Title': 'Title'
-    })
-    file_out = open(output_name, 'wb')
-    pdf_merger.write(file_out)
-
-    file_in.close()
+    with open(file_name, "rb") as file_in, open(output_name, "wb") as file_out:
```

Comment:
Always use context managers to handle open files, connections, sessions etc. It will take care of shuting down resources in case of an error.

Reply:
@mirmozavr absolutely! guess, remember, I copied this piece directly from SO 😅